internal/core/metadata/v2/controller/http/deviceprofile.go

```diff
@@ -131,3 +130,29 @@ func (dc *DeviceProfileController) AddDeviceProfileByYaml(w http.ResponseWriter,
 	// Encode and send the resp body as JSON format
 	pkg.Encode(addDeviceProfileResponse, w, lc)
 }
+
+func (dc *DeviceProfileController) GetDeviceProfileByName(w http.ResponseWriter, r *http.Request) {
+	lc := container.LoggingClientFrom(dc.dic.Get)
+	ctx := r.Context()
+
+	// URL parameters
+	vars := mux.Vars(r)
+	name := vars[v2.Name]
+
+	var response interface{}
+	var statusCode int
+
+	deviceProfile, err := application.GetDeviceProfileByName(name, ctx, dc.dic)
+	if err != nil {
+		lc.Error(err.Error())
```

Comment:
the error message should not be logged if it is errors.KindEntityDoesNotExist (not found)

Reply:
Modified, log error message when errors.Kind(err) != errors.KindEntityDoesNotExist